apps/manager_app/onboarding/onboarding.h

```diff
@@ -24,6 +24,14 @@
 /*****************************************************************************
  * TYPEDEFS
  *****************************************************************************/
+typedef enum {
+  ONBOARDING_VIRGIN_DEVICE,
+  ONBOARDING_DEVICE_AUTH,
+  ONBOARDING_JOYSTICK_TRAINING,
+  ONBOARDING_CARD_CHECKUP,
+  ONBOARDING_CARD_AUTHENTICATION,
```

Comment:
Need to find a way to set card auth done in onboarding. Or rely on CySync to directly set the onboarding complete flow.

Reply:
We will take it up when we implement card tap check/ card authentication. Let's keep this out of scope from this pr